bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+of the centralized trading platform. While decentralized exchanges follow the concept "the trade is the settlement", we
+here propose to let trades happen outside the blockchain while settlements are performed regularly on-chain. 
+
+## Savings Account
+
+Managed vesting balances furthermore allow to setup *savings accounts* (or rather *savings balances*) which are balances
+that can only be accessed after a certain amount of time has elapsed (manager can be set optional).
+
+# Use-case off-chain activities
+
+We propose to extend the BitShares Blockchain in that each account is provided an additional freedom and possibilities 
+in configurating their vesting balances with locked up funds.
+
+## Exchange perspective
+
+### Customer sign up
+When a new customer signs up with the centralized, external exchange (CEX), a BitShares account must be uniquely linked. This can be done e.g. with a signed message. 
+
+### Customer on-boarding
+The centralized, external exchange (CEX) must monitor all vesting balances that are created on the BitShares Blockchain.
+If a managed vesting balance is recognized that has this CEX as settlement authority and otherwise matching 
+parameters (delay time period, cliff period), the asset and amount are credited to the customers account on the CEX.
+
+The customer could have the option to either 
+1. have the CEX create everything (BitShares account and vesting balances), CEX must provide the customer with the private keys in a safe manner
+2. create the BitShares account by itself, link it to the CEX account and let the CEX create the vesting balances
+3. create the BitShares account and vesting balance itself, signup at CEX and linking of accounts can also happen afterwards
+
+Once a vesting balance is created, the customer can charge it with a new operation introduced in specifications.
+
+### Customer off-boarding
+The customer initiates a withdrawal and must wait a predefined time period (delay). After this has passed, the funds 
+are automatically transfered to the customer's account. The CEX must monitor all vesting balances, in particular 
+which balances have a pending withdrawal. If a customers initiates a withdrawal of funds that are still in its vesting 
+balance on the BitShares Blockchain, but has already traded on the CEX, the CEX must settle those vesting balances before 
+the withdrawal executes, to prevent loss on the CEX side.
+After a complete withdrawal the vesting balance object will exist for a while before being deleted. Effectively the customer stays a customer of the CEX (at least on chain), only with an empty balance.
+
+### Trading
+After on-boarding the CEX knows that it can perform **settlements** of the corresponding balances and thus
+can allow the customer to trade with those funds in the CEX internal markets, let's call that *trading balance* for the CEX.
+
+To compare this with regular centralized exchanges, withdrawals and deposits are performed against the trading balance.
```

Comment:
If I understand the intent correctly, I think that this might be better phrased as "In contrast to regular centralized exchanges, ..."

Reply:
The wording is correct, as already existing CEX have trading balances in the same manner.